Dockerfile

```diff
@@ -1,20 +1,26 @@
 ARG DOCKER_PLATFORM
 ARG DOCKER_ARCH
 
-FROM --platform=${DOCKER_PLATFORM:-linux/amd64} ${DOCKER_ARCH:-amd64}/debian:buster-slim
+FROM --platform=${DOCKER_PLATFORM:-linux/amd64} ${DOCKER_ARCH:-amd64}/debian:bookworm-slim
 ARG GOARCH
 ARG VERSION_SERVER_URL
 ARG SUPERVISOR_SERVER
-ENV GOARCH ${GOARCH:-amd64}
-ENV VERSION_SERVER_URL ${VERSION_SERVER_URL:-https://version.storj.io}
-ENV SUPERVISOR_SERVER ${SUPERVISOR_SERVER:-unix}
+ENV GOARCH=${GOARCH:-amd64}
+ENV VERSION_SERVER_URL=${VERSION_SERVER_URL:-https://version.storj.io}
+ENV SUPERVISOR_SERVER=${SUPERVISOR_SERVER:-unix}
 
 RUN apt-get update
 RUN apt-get install -y --no-install-recommends ca-certificates supervisor unzip wget
 RUN update-ca-certificates
 
 RUN mkdir -p /var/log/supervisor /app
-COPY docker/ /
```

Comment:
We can keep this.

The difference between Debian _buster_ and _bookworm_ is that following the conventions that recent Linux distributions adopted `/bin` is a symlink to `/usr/bin` and _Docker_ `COPY` won't copy `docker/bin` over `/bin` symbolic link.

Your solution works, but we'll need to update the docker image if we add more files.

The change that we have to do is to move `docker/bin` to `docker/usr/bin`.

__Facts__

`ls -la` on Debian _buster_
![image](https://github.com/user-attachments/assets/b0b4c7b1-170f-422c-8617-39b84d4e41b2)

`ls -la` on Debian _bookworm_
![image](https://github.com/user-attachments/assets/421247ee-f329-4ee9-8ad0-a0b3887f6386)

Reply:
I addressed the issue, if possible, I'm dying to know how do you debug that.